File: .taira-source-c80bf4d85a97/java/iroha_android/src/main/java/org/hyperledger/iroha/android/subscriptions/SubscriptionToriiException.java
package org.hyperledger.iroha.android.subscriptions;

/** Runtime error raised when subscription Torii endpoints reject a request or parsing fails. */
public final class SubscriptionToriiException extends RuntimeException {

  private static final long serialVersionUID = 1L;

  public SubscriptionToriiException(final String message) {
    super(message);
  }

  public SubscriptionToriiException(final String message, final Throwable cause) {
    super(message, cause);
  }
}

File: .taira-source-c80bf4d85a97/kotlin/core-jvm/src/main/java/org/hyperledger/iroha/sdk/subscriptions/SubscriptionToriiException.kt
package org.hyperledger.iroha.sdk.subscriptions

/** Runtime error raised when subscription Torii endpoints reject a request or parsing fails. */
class SubscriptionToriiException : RuntimeException {
    constructor(message: String) : super(message)
    constructor(message: String, cause: Throwable) : super(message, cause)
}
